test-apps/display-test-app/src/frontend/TextDecoration.ts

```diff
@@ -115,7 +119,7 @@ class TextEditor implements Decorator {
       toleranceLog10: -5,
       type: "2d",
       placement: {
-        origin: this.origin.toJSON(),
+        origin: Point3d.createZero(),
```

Comment:
Presumably changed to remove this as a confounding factor. Revert when fixed.

Reply:
I set the origin in the text block instead, only because that more closely matches what is needed when text is used within a more complex annotation element like a dimension.